ports/unix/Makefile

```diff
@@ -175,6 +177,15 @@ SRC_QSTR += $(SRC_C) $(LIB_SRC_C)
 # SRC_QSTR
 SRC_QSTR_AUTO_DEPS +=
 
+ifneq ($(FROZEN_MANIFEST),)
```

Comment:
Can this be OR'ed into the next section?

Reply:
Done (with trick, because makefile doesn't have OR).